### ISSUE TITLE
Failed to download attachment using keyboard navigation

### ISSUE THREAD

**Issue by amottier** [gristlabs/grist-core#2150]
### Describe the current behavior

When using the keyboard tab key to highlight the "download" button and using space key to trigger the action nothing happen. Doing the same and highlighting the "delete" button allow me to actually delete the attachment.

### Steps to reproduce

1. Create a new document and set the column A type to "Attachment"
2. Add an attachment to the first row
3. Use Enter key to open the attachement
4. Use tab key to navigate to the "Download" button (first hit will highlight the attachment file name, second one the download button)
5. Type space key to start the download → nothing happen, hit tab key twice to highlight "Delete" button and hit space key → the attachment is deleted.

### Describe the expected behavior

It should be possible to trigger the download action using keyboard.

### Where have you encountered this bug?

- [x] On [docs.getgrist.com](https://docs.getgrist.com)
- [x] On a self-hosted instance

### Instance information (when self-hosting only)

* Grist instance:
  * Version: 1.7.11
  * Installation mode: Docker
  * Architecture: single-worker

* This has been tested on Firefox 140.8.0esr (package provided in Debian 13).
* Link to browser console log if relevant: no log in console
* Link to server log if relevant: no log in server

**Comment by amottier** [comment]
A small ![video](https://github.com/user-attachments/assets/91eb0fce-eedb-41a0-b068-62949550fb2d) to illustrate the issue.
